parcels/kernel.py

```diff
@@ -230,6 +235,11 @@ def execute_python(self, pset, endtime, dt):
         # back up variables in case of ErrorCode.Repeat
         p_var_back = {}
 
+        for f in self.fieldset.get_fields():
+            if type(f) in [VectorField, NestedField, SummedField]:
+                continue
+            f.data = np.array(f.data)
```

Comment:
I guess that in the future we might also do something smarter with chunking in SciPy mode here? Because now we're just moving the whole field to a numpy array?